docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -2,16 +2,21 @@
 
 = Deploying a {Ceph} ingress daemon
 
-To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
-`HAProxy` is managed by {OpenStackPreviousInstaller} through `Pacemaker`: the three running instances at this point will point to the old RGW backends, resulting in a broken configuration.
+To ensure that you distribute the correct number of services to your available {Ceph} nodes, you use `cephadm` labels to refer to a group of nodes where a given daemon type is deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
+
+`HAProxy` is managed by {OpenStackPreviousInstaller} through Pacemaker. The three running instances are pointing to the old RGW back ends, resulting in a broken configuration.
+
```

Comment:
I’m not really sure how the first two paragraphs relate to the procedure. We also already said the first paragraph in the “Migrating the Ceph RGW back ends” procedure intro. To me, the description of the procedure is the part that starts with “To deploy the Ceph ingress daemon…” Can you help clarify the intent of the procedure?

Reply:
Correct, this sounds redundant in this case. I think the goal was to highlight that when we deploy the Ingress daemon, two new containers are created:
1. haproxy, used to reach the backends
2. keepalived, used to own the VIP

if those containers are deployed on two nodes (it's the minimum requirement for HA), we have 4 instances of them, and they take their role in the cardinality picture (that is supposed to limit the amount of services in terms of containers per node).
We can rework the above ^ and replace these sentences, or remove them entirely and simply put a reference to the cardinality section.
However I think it's good to clarify this aspect to customers, even with a single sentence that explains the implications of the next steps. 
@klgill wdyt?

Reply:
@fmount I suggest that you review the latest iteration of this introduction and let me know if it is saying what we want to say. 
I have some follow-up questions about your comment above:

1. Will those containers always be deployed on two nodes automatically, or does the user choose how to distribute the containers among their nodes? 
2. 4 instances = 2 HAProxy containers and 2 Keepalived containers across two nodes?
3. I’m not sure what is meant by “they take their role in the cardinality picture.” Specifically, “take their role” is unclear to me.

Reply:
Hi @klgill, 
not sure I missed this, but in general:
1. they're are deployed on 2 nodes automatically, but  given we put the `rgw` label, we can control where they land;
2. correct, this is needed for HA and VIP failover;
3. it means they are additional containers deployed on a node where we need to make sure to not co-locate much daemons: however, it's not obvious that Ingress daemon means 2 more containers on two different nodes, and this is why they play a role in the cardinality computation.